specification/netapp/resource-manager/Microsoft.NetApp/stable/2025-06-01/netapp.json

```diff
@@ -5473,5132 +7496,4967 @@
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
         },
-        "nextLink": {
-          "type": "string",
-          "description": "URL to get the next set of results."
-        }
-      }
-    },
-    "UsageResult": {
-      "description": "Usages entity model",
-      "type": "object",
-      "properties": {
         "id": {
-          "description": "The id of the usage.",
-          "readOnly": true,
-          "type": "string"
+          "type": "string",
+          "description": "Resource Id",
+          "readOnly": true
         },
         "name": {
-          "description": "The name of the usage.",
-          "$ref": "#/definitions/usageName"
-        },
-        "properties": {
-          "description": "Usage properties",
-          "$ref": "#/definitions/usageProperties",
-          "x-ms-client-flatten": true
-        }
-      }
-    },
-    "usageProperties": {
-      "description": "Usage properties",
-      "type": "object",
-      "properties": {
-        "currentValue": {
-          "description": "The current usage value for the subscription.",
-          "readOnly": true,
-          "type": "integer",
-          "format": "int32"
-        },
-        "limit": {
-          "description": "The limit of the usage.",
-          "readOnly": true,
-          "type": "integer",
-          "format": "int32"
+          "type": "string",
+          "description": "Resource name",
+          "readOnly": true
         },
-        "unit": {
-          "description": "The unit of the usage.",
-          "readOnly": true,
-          "type": "string"
-        }
-      }
-    },
-    "usageName": {
-      "description": "The name of the usage.",
-      "type": "object",
-      "readOnly": true,
-      "properties": {
-        "value": {
-          "description": "The name of the usage.",
-          "type": "string"
+        "type": {
+          "type": "string",
+          "description": "Resource type",
+          "readOnly": true
         },
-        "localizedValue": {
-          "description": "The localized name of the usage.",
-          "type": "string"
-        }
-      }
-    },
-    "netAppAccountList": {
-      "description": "List of NetApp account resources",
-      "type": "object",
-      "properties": {
-        "value": {
-          "type": "array",
-          "description": "Multiple NetApp accounts",
-          "items": {
-            "$ref": "#/definitions/netAppAccount"
+        "tags": {
+          "type": "object",
+          "description": "Resource tags",
+          "additionalProperties": {
+            "type": "string"
           }
         },
-        "nextLink": {
-          "type": "string",
-          "description": "URL to get the next set of results."
-        }
-      }
-    },
-    "netAppAccount": {
-      "description": "NetApp account resource",
-      "type": "object",
-      "required": [
-        "location"
-      ],
```

Comment:
`location` as "required" written this way does not have effect, as the property is in parent.

TypeSpec should have `location` as "required", as in parent
https://github.com/Azure/typespec-azure/blob/a191aa2/packages/typespec-azure-resource-manager/lib/common-types/types.tsp#L48